<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormLogin</class>
 <widget class="QWidget" name="FormLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>chat login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/logo.png</normaloff>icons/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#FormLogin{
 background-image: url(icons/bg1_1.jpg);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>input your name:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="userName">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>80</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="img">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>-3</y>
     <width>421</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <zorder>img</zorder>
  <zorder>label</zorder>
  <zorder>userName</zorder>
  <zorder>loginBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
